### ISSUE TITLE
Parser error when exporting variable with comment between identifier and =

### ISSUE THREAD

**Issue by zaygraveyard** [guybedford/es-module-lexer#34]
```html
<output id="log"></output>
<script type="module">
import {parse} from 'https://unpkg.com/es-module-lexer@0.3.12?module';

(async function() {
	try {
		await parse('export var a/**/=0;');
		window.log.textContent = 'No errors';
	} catch (error) {
		window.log.textContent = error.message;
	}
})();
</script>
```
Try it online https://jsfiddle.net/679cep3n/1/

**Comment by guybedford** [comment]
Thanks for posting this case, I've added a fix in https://github.com/guybedford/es-module-lexer/pull/35. Will wait a little while before releasing though.

**Comment by guybedford** [comment]
Released in 0.3.13.

**Comment by zaygraveyard** [comment]
Thank you 😄
